build/getDependencies-windows.sh

```diff
@@ -109,7 +109,7 @@ rm -rf ../src/BloomBrowserUI/bookEdit/test/libsynphony
 #     clean: false
 #     revision: latest.lastSuccessful
 #     paths: {"Palaso.BuildTasks.dll"=>"build/", "*.dll"=>"lib/dotnet"}
-#     VCS: https://github.com/sillsdev/libpalaso.git [NoStrongName]
```

Comment:
Only a comment, anyway, I see...so the change that caused it must already have happened, presumably without hurting us. Let's not worry about it.